lib/index.js

```diff
@@ -47,10 +47,10 @@ function NwBuilder(options) {
         argv: process.argv.slice(2)
     };
     // Intercept the platforms and check for the legacy platforms of 'osx' and 'win' and
-    // replace with 'osx32', 'osx64', and 'win32', 'win64' respectively.
+    // replace with 'osx64', and 'win32', 'win64' respectively.
     if(typeof options.platforms != 'undefined'){
         if(options.platforms.indexOf('osx') >= 0){
-            options.platforms.splice(options.platforms.indexOf('osx'), 1, 'osx32', 'osx64');
+            options.platforms.splice(options.platforms.indexOf('osx'), 1, 'osx64');
```

Comment:
Sorry now, I'm just looking at this again and thinking this is actually a breaking change. I.e. it would break people's builds relying on it. So if it was merged, it would require a major version bump (to 4.0.0), which would then require people to update their package.json / reinstall. I'm not sure this is worth it. Maybe we could not change this part (`osx` -> `osx32`, `osx64`), merge the rest, and wait until a 4.0.0 to include this specific change. What do you think?